chatformatter-core/src/main/java/com/eternalcode/formatter/config/PluginConfig.java

```diff
@@ -25,7 +28,7 @@ public class PluginConfig implements ChatSettings, PlaceholderStack, TemplateRep
     public boolean receiveUpdates = true;
 
 
-    @Description({ " ", "# Chat format for ranks (Vault) Support mini-messages and legacy colors" })
+    @Description({ " ", "# Chat rankFormat for ranks (Vault) Support mini-messages and legacy colors" })
     @Description({ " ", "# We're recommending to use webui for mini-messages: https://webui.adventure.kyori.net/" })
     @Description({ " ", "# documentation is here: https://docs.adventure.kyori.net/minimessage/format.html" })
 
```

Comment:
`@Description({ " ", "# You can check LuckyPerms setup and placeholders here: https://luckperms.net/wiki/Placeholders" })`